app/models/katello/rhsm_fact_parser.rb

```diff
@@ -79,6 +79,22 @@ def boot_timestamp
       facts['proc_stat.btime']&.to_i
     end
 
+    def virtual
+      facts['virt.is_guest']
+    end
+
+    def ram
+      facts['memory.memtotal']
```

Comment:
this may need to be *1024 if we use `to_s.to_gb` in core PR

Reply:
if this fact is coming from RHSM, shouldn't it be `/ 1024`? I have `memory.memtotal: 990712
` on my system.

There is also `dmi.memory.size`, which gives the value in human-readable format.

Reply:
the core PR meanwhile changed to store the number in MB, so I agree, this should be 1024 now